apps/vast/vast_threatbus/vast_threatbus.py

```diff
@@ -362,6 +362,8 @@ async def retro_match_vast(
         start = time.time()
         vast = VAST(binary=vast_binary, endpoint=vast_endpoint, logger=logger)
         kwargs = {}
+        # Don't allocate unnecessary resources.
+        kwargs["caf.scheduler.max_threads"] = 1
```

Comment:
```suggestion
        kwargs["caf.scheduler.max-threads"] = 1
```

Didn't setting this to 1 cause an infinite hang, which is why it defaults to 2 for client processes?

This would also require a new release.

Reply:
`vast import` requires 2, but `vast export` works with 1, at least when I tested it on Linux.

Reply:
Regarding the suggested change: Look at other options we submit. Afair pyvast doesn't handle dashes correctly for some implementation detail reason, so it rewrites underscores to dashes instead. See https://github.com/tenzir/vast/blob/master/pyvast/pyvast/vast.py#L83-L91.

Reply:
That makes sense. Thanks!